### ISSUE TITLE
nightly build broken.

### ISSUE THREAD

**Issue by hainm** [Amber-MD/cmake-buildscripts#79]
Please see here

https://travis-ci.org/Amber-MD/ambertools-ci/jobs/288105255
```
/usr/bin/ld: cannot find -lTERMCAP_LIBRARY
collect2: error: ld returned 1 exit status
make[2]: *** [AmberTools/src/readline/libreadline.so] Error 1
make[1]: *** [AmberTools/src/readline/CMakeFiles/readline.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by hainm** [comment]
and here: https://travis-ci.org/Amber-MD/ambertools-ci/jobs/288105256

**Comment by multiplemonomials** [comment]
Should be fixed by commit d1d3b87
